src/content/docs/release-notes/docs-release-notes/docs-9-13-2024.mdx

```diff
@@ -0,0 +1,68 @@
+---
+subject: "Docs"
+releaseDate: '2024-09-13'
+version: 'version: September 6-12, 2024'
+---
+
+### New docs
+
+* [AWS Application Migration Service integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-application-migration-services-integration) describes how to monitor migrations you've migrated to AWS.
```

Comment:
This isn't a super big deal but "Migration Service integration describes how to monitor migrations you've migrated" is a lot of "migrate" usage!

Reply:
You're so right! I have no one to blame but myself :D